neuromation/cli/job.py

```diff
@@ -877,6 +873,55 @@ def format_fail(job: str, reason: Exception) -> str:
     return job
 
 
+async def _build_volumes(
+    root: Root, input_volumes: Sequence[str], env_dict: Dict[str, str]
```

Comment:
are `input_volumes` unique?

Reply:
no, it's direct user input.

Reply:
should we handle edge cases such as same volumes etc? (prob out of scope)

Reply:
looking at the code, I can say that this case is handled properly: the result collection `volumes` is of type `Set` (but in order to reduce number of loop iterations, I changed input array to the type of `Set` as well). 

However, this is a good case indeed, `neuro run --volume=ALL --volume=<custom>`, so since `--volume=ALL` already mounts *all possible* volumes, I changed the code to forbid this option together with any other option -- thank you for pointing this out!